br/pkg/storage/gcs.go

```diff
@@ -431,7 +431,7 @@ func (s *GCSStorage) Reset(ctx context.Context) error {
 			if err != nil {
 				return errors.Trace(err)
 			}
-			client.SetRetry(storage.WithErrorFunc(shouldRetry))
+			client.SetRetry(storage.WithErrorFunc(shouldRetry), storage.WithPolicy(storage.RetryAlways))
```

Comment:
`storage.WithErrorFunc(shouldRetry)` is used for retry for white list errors, why set it to always?

Reply:
seems irrelevant
```
// RetryPolicy describes the available policies for which operations should be
// retried. The default is `RetryIdempotent`.
type RetryPolicy int

```